bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/rest/MediaReceiverManager.java

```diff
@@ -68,15 +68,16 @@ public MediaReceiverManager(FreeboxOsSession session, UriBuilder uriBuilder) thr
 
     public void sendToReceiver(String receiver, String password, Action action, MediaType type)
             throws FreeboxException {
-        sendToReceiver(receiver, new Request(password, action, type, null, 0));
+        sendToReceiver(receiver,
+                new Request(password, action.name().toLowerCase(), type.name().toLowerCase(), null, 0));
     }
 
     public void sendToReceiver(String receiver, String password, Action action, MediaType type, String url)
             throws FreeboxException {
-        sendToReceiver(receiver, new Request(password, action, type, url, 0));
+        sendToReceiver(receiver, new Request(password, action.name().toLowerCase(), type.name().toLowerCase(), url, 0));
```

Comment:
Shouldn't the type casting `'.name().toLowerCase()'` be done directly in `Request` class ?

Reply:
Done